Triggers and fixes

olemyk opened the pull request:
Created triggers for Scale_prepare for
Disable SELinux, Enable SSH root login, Enable SSH pubkey authentication, Disable SSH hostkey checking
Default is true.

Rerunn of playbooks ends in that MSG: Incorrect type for fail_msg or msg, expected string and got <type 'list'>
"assert msg no longer accepts list #48547 Another option to configure this is to add stdout_callback = debug to your ansible.cfg"
Change the msg under "update | Check if any running node needs to be updated" remove the split for "msg"

There was some error for installation of RPM:
Error: Invoking "yum" only once while using a loop via squash_actions is deprecated. Instead of using a loop to supply multiple items and specifying `name:
Change "install | Install GPFS RPMs"  "build | Install prereqs for building GPL module from source" to use name insted of with_item.

olemyk committed:
Merge pull request #1 from acch/master

new function

olemyk committed:
Created triggers for Scale_prepare for
Disable SELinux, Enable SSH root login, Enable SSH pubkey authentication, Disable SSH hostkey checking
Default is true.

Rerunn of playbooks ends in that MSG: Incorrect type for fail_msg or msg, expected string and got <type 'list'>
"assert msg no longer accepts list #48547 Another option to configure this is to add stdout_callback = debug to your ansible.cfg" to get multiline
Change the msg under "update | Check if any running node needs to be updated" remove the split for "msg"

Added Current scale and available version in message banner

There was some error for installation of RPM:
Error: Invoking "yum" only once while using a loop via squash_actions is deprecated. Instead of using a loop to supply multiple items and specifying `name:
Change "install | Install GPFS RPMs"  "build | Install prereqs for building GPL module from source" to use name insted of with_item.

acch commented:
Awsome - thank you very much! I'm back in the office next week and will test your changes.

BTW: Which ansible version did you test this with?

acch commented:
One question though: I see that you've defined `scale_prepare_secure_environment` but you don't seem to pick it up (use it) anywhere... Am I missing something? If not then I'd suggest to remove it from `defaults/main.yml` for the time being.

olemyk commented:
Should be lates version of ansible 2.7. 

Scale_prepare_secure_environment is a todo item. So yes, we could remove It and add it later.

olemyk committed:
Created triggers for Scale_prepare for
Disable SELinux, Enable SSH root login, Enable SSH pubkey authentication, Disable SSH hostkey checking
Default is true.

Rerunn of playbooks ends in that MSG: Incorrect type for fail_msg or msg, expected string and got <type 'list'>
"assert msg no longer accepts list #48547 Another option to configure this is to add stdout_callback = debug to your ansible.cfg" to get multiline
Change the msg under "update | Check if any running node needs to be updated" remove the split for "msg"

Added Current scale and available version in message banner

There was some error for installation of RPM:
Error: Invoking "yum" only once while using a loop via squash_actions is deprecated. Instead of using a loop to supply multiple items and specifying `name:
Change "install | Install GPFS RPMs"  "build | Install prereqs for building GPL module from source" to use name insted of with_item.

acch commented:
Hey there - finally found the time to look into this. I've merged your changes - thanks again very much for your contribution!

I've moved another "TODO comment", as well another warning I'm still getting with ansible 2.7.8 to issue #4.

I've added some further (minor) changes so make sure to pull master... Cheers and all the best!
